languages/ruby/exercises/concept/conditionals-ternary/moviegoer.rb

```diff
@@ -0,0 +1,22 @@
+class Moviegoer
+  attr_reader :age, :movie_club_member
+
+  def initialize(age, movie_club_member: false)
```

Comment:
You are using a default value here, there are two options:

1) Make sure this exercise depends on students already knowing what a default value is, so maybe it needs to depend on the `default-values` concept.
2) Does it need a default value? While yes, we might usually use one in our own code, perhaps we don't need it to illustrate this concept?

Reply:
Also, the idea of a keyword argument should be a prerequisite to this exercise, so we might want to ensure that.

Reply:
Or if it is not covered yet, then we would want a default argument, which should then be the prerequisite.

Reply:
Nice catch @kotp. Keyword args are actually introduced in basics so we don't need to teach those. But we could introduce default-values in this exercise as well?

Reply:
If we are introducing default values, then perhaps `nil` is suitable as the default value by default.  `false` probably being a better choice for various reasons.

The concept of "ternary" alone seems that it might be very light.

Reply:
Also a good suggestion! But [Instance variables teaches nil](https://github.com/exercism/v3/blob/master/languages/ruby/exercises/concept/instance-variables/.docs/introduction.md) already 🙂

Reply:
Kind of, but not default arguments, surely not default keyword arguments.

Reply:
I also think we are in for some churn shortly as well with this in regards to Ruby with 3.0 around the corner.